src/cmd-build

```diff
@@ -440,6 +440,12 @@ else
                  "--label=io.openshift.build.versions=machine-os=${buildid}"
                 )
     fi
+    previous_build=$(get_latest_build)
```

Comment:
Note we already have a `previous_build` variable defined like this.

But I think what we probably actually want for this is the parent build? (With a fallback to the previous build I suppose.) The parent build is the last build that was released to users. It's used in FCOS only today, but could be used by RHCOS too.